app/models/attachment.rb

```diff
@@ -25,4 +26,9 @@ def update_attributes(new_params)
   def changed?
     @changed
   end
+
+  def self.all_from_publishing_api(payload)
```

Comment:
Yeah, we originally had it as `from_publishing_api` but thought it needed something to differentiate because it would extract all the attachments, rather than just one.